### PR TITLE
feat: input group for checkbox/radio inline

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles#62

## Description
Remove form-item--inline and add form-group--inline class

Nothing has visually changed but because the visual tests for inline elements have moved to the form-group level they are now in the form-group-visual regression tests

## Breaking Change
* Remove `fd-form-item--inline` class
* Add `fd-form-group--inline` class, which is parent of multiple `fd-form-item`s

### CODE REVIEW

File: src/form-item.scss [hunk 1]
```diff
@@ -27,9 +27,9 @@ $block: #{$fd-namespace}-form-item;
     }
   }
 
-  &--inline {
-    display: block;
-    float: left;
-    margin-right: fd-space(4);
-  }
+  // &--inline {
```
Comment (jbadan) [inline src/form-item.scss]: Can we just delete this?

File: test/templates/form-group/index.njk [hunk 2]
```diff
@@ -27,4 +27,56 @@
   {% endset %}
   {{ format(example) }}
 
+  <h2>Inline radio</h2>
+
+  {% set example %}
+    <fieldset class="fd-fieldset">
+      <legend class="fd-fieldset__legend">Inline Radio buttons</legend>
+      <div class="fd-form-group--inline">
+          <div class="fd-form-item fd-form-item">
```
Comment (jbadan) [inline test/templates/form-group/index.njk]: this example has double `fd-form-item` on each div

**Comment by InnaAtanasova** [comment]
`test/templates/form-group/index.njk` still has double `fd-form-item` on each div

File: docs/pages/components/form.md [hunk 3]
```diff
@@ -92,25 +92,25 @@ Do not use the input field if:
...
     <br />
-    <div class="fd-form-item fd-form-item--horizontal">
+    <div class="fd-form-item--horizontal">
         <label class="fd-form-label" for="input-1a">Default Input:</label>
         <input class="fd-input" type="text" id="input-1a" placeholder="Field placeholder text">
     </div>
     <br />
     <br />
-    <div class="fd-form-item fd-form-item--horizontal">
+    <div class="fd-form-item--horizontal">
```
Comment (JKMarkowski) [inline docs/pages/components/form.md]: Is it good usage of BEM things? I see `fd-form-item` gives `fd-reset()`

File: src/form-group.scss [hunk 4]
```diff
@@ -17,3 +17,12 @@ $block: #{$fd-namespace}-form-group;
     margin-bottom: 0;
   }
 }
+
+.#{$block}--inline {
+  display: flex;
+
+  .fd-form-item {
+    float: left;
```
Comment (JKMarkowski) [inline src/form-group.scss]: Shouldn't we avoid floats generally ?

File: docs/pages/components/form.md (hunk 3 above)
Comment (JKMarkowski) [inline docs/pages/components/form.md]: Is it good usage of BEM things? I see `fd-form-item` gives `fd-reset()` Normally it should be `fd-form-item fd-form-item--horizontal`

File: test/templates/form-group/index.njk [hunk 5]
```diff
@@ -27,4 +27,56 @@
...
+{% endset %}
+{{ format(example) }}
+<br>
+
+  <h2>Checkbox inline </h2>
+
+  {% set example %}
+    <fieldset class="fd-fieldset">
+        <legend class="fd-fieldset__legend">Checkboxes inline</legend>
+        <div class="fd-form-group--inline">
```
Comment (InnaAtanasova) [inline test/templates/form-group/index.njk]: Should be `fd-form-group fd-form-group--inline`. `fd-form-group` brings the reset

File: src/form-group.scss [hunk 6]
```diff
@@ -17,3 +17,12 @@ $block: #{$fd-namespace}-form-group;
...
   }
 }
+
+.#{$block}--inline {
+  display: flex;
+  justify-content: flex-start;
+
+  > :not(:last-child) {
+    margin-right: fd-space(4);
+  }
```
Comment (InnaAtanasova) [inline src/form-group.scss]: This is scss, you can use nesting to specify that an element is a child of another element or `&`

**Pull request merged**